components/html-block/demo/html-block.html

```diff
@@ -41,6 +52,7 @@ <h6>heading 6</h6>
 								<li>ordered item 2</li>
 							</ol>
 							<div><a href="https://d2l.com">anchor</a></div>
+							<div style="display: none">\( )\</div>
```

Comment:
Is this needed?

Reply:
So the first HTML block that renders always wins, which means if this first HTML block doesn't contain LaTeX, we'll load the wrong MathJax renderer. This was mostly just a quick and dirty way to achieve that without placing the math examples first. Feels like it would also be fine to just only display the LaTeX examples in the `latex=true` case and omit them in the base case.